src/rust/engine/process_execution/src/remote.rs

```diff
@@ -61,15 +67,18 @@ impl CommandRunner {
     stream
         .take(1)
         .into_future()
+        // If there was a response, drop the _stream to disconnect so that the server doesn't keep
+        // the connection alive and continue sending on it.
+        .map(|(maybe_operation, _stream)| maybe_operation)
```

Comment:
Worth explicitly dropping with `mem::drop` and placing the comment there? On the other hand, maybe doing it this way is less confusing for a new rust user because it doesn't make `mem::drop` seem magical.

Reply:
Done